biome.json

```diff
@@ -5,8 +5,7 @@
     "ignore": ["example", "expo-example", "docs", "plugin", "lib"]
   },
   "formatter": {
-    "enabled": true,
-    "indentStyle": "tab"
+    "enabled": false
```

Comment:
I tested it locally, and now we can enable biome check as it returned 39 errors about import order

Reply:
Nice, I was gonna ask you about it. Run it on the latest commit